nimare/meta/ibma.py

```diff
@@ -8,7 +8,7 @@
 import pandas as pd
 import pymare
 from joblib import Memory
-from nilearn._utils.niimg_conversions import _check_same_fov
+from nilearn._utils.niimg_conversions import check_same_fov
```

Comment:
if we want to continue to support 0.10.1 nilearn
```suggestion
try:
    # nilearn>=0.10.3
     from nilearn._utils.niimg_conversions import check_same_fov
except ImportError:
    # nilearn < 0.10.3
    from nilearn._utils.niimg_conversions import _check_same_fov as check_same_fov
```